app/models/condition.rb

```diff
@@ -36,6 +37,17 @@ def warning_answer_doesnt_exist
     { name: "answer_value_doesnt_exist" }
   end
 
+  def warning_routing_to_next_page
+    return nil if check_page.nil? || goto_page.nil?
+
+    routing_page_position = check_page.position
+    goto_page_position = goto_page.position
+
+    return { name: "cannot_route_to_next_page" } if goto_page_position == (routing_page_position + 1)
```

Comment:
Do we have a different warning that we'll use for routing to the CYA from the final page? It's possible that we could incorporate that check into this warning - or build another warning that can handle stuff to do with CYA routes

Reply:
Good question, I think this is something we will need to pick up when we add CYA as an option to the goto pages. CYA is very different from other pages as the concept doesn't really exist in our data model, this is why I think we need to pick it up as another ticket.